### PR TITLE
reworked on docu

### PR DESCRIPTION
**Pull request by JefMari**
image
image

### CODE REVIEW

File: docs/sections/Docs-Forms/index.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default class Forms extends React.Component {
         <p className="lead">Style individual form controls and utilize common layouts.</p>
 
         <section className="doc-bottom-space-large">
-          <section className="doc-bottom-space">
+          <section className="doc-bottom-space-largem-space">
```
Comment (brianbodollo) [inline docs/sections/Docs-Forms/index.jsx]: do we have `doc-bottom-space-largem-space`?

Comment (srph) [inline docs/sections/Docs-Forms/index.jsx]: typo

**Comment by brianbodollo** [comment]
loooooooook soo good!!

File: docs/sections/Docs-Forms/index.jsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ export default class Forms extends React.Component {
 
         <section className="doc-bottom-space-large">
           <h3 className="doc-heading">Form Groups</h3>
-          <p>Used as a container for a form element.</p>
+          <p>Used <code>.form-group</code> as a container for form elements to have an ideal spacing.</p>
```
Comment (srph) [inline docs/sections/Docs-Forms/index.jsx:79]: _Use_, you mean?

File: docs/sections/Docs-Forms/index.jsx [hunk 3]
```diff
@@ -93,7 +93,7 @@ export default class Forms extends React.Component {
         <section>
           <section className="doc-bottom-space">
             <h3 className="doc-heading">Input Groups</h3>
-            <p>Used as a container for a form element.</p>
+            <p>Used <code>.input-group</code> as a container for a group of input elements.</p>
```
Comment (srph) [inline docs/sections/Docs-Forms/index.jsx:96]: _Use_, you mean?

**Pull request merged**